recipe/meta.yaml

```diff
@@ -47,12 +47,14 @@ requirements:
     - {{ compiler('cuda') }}
     - sysroot_linux-64 2.17  # [linux64]
     - libgomp  # [linux]
-    - 7zip  # [win]
+  host:
```

Comment:
@isuruf Why do we need an empty host section?

Reply:
Historically conda-build did not separate `build` in `host` (in fact there was only `build` originally). Listing a separate empty `host` ensures conda-build does split them (though usually it is not needed)